tests/e2e/test_alembic_packaging.py

```
@@ -0,0 +1,177 @@
+    run_dir = tmp_path / "run"
+    run_dir.mkdir()
+    script = (
+        "import asyncio\n"
+        "from cloud_agents.storage.run_state_store import RunStateStore\n"
+        "async def main():\n"
+        f"    store = RunStateStore(db_url={throwaway_db_url!r})\n"
+        "    await store.connect()\n"
+        "    await store.close()\n"
+        "asyncio.run(main())\n"
+    )
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)